.github/workflows/test.yml

```diff
@@ -0,0 +1,59 @@
+# Heavily based on https://joelhooks.com/jest-and-github-actions
+name: Tests CI
+
+on:
+  pull_request:
+    paths-ignore:
+        - 'README.md'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Reconfigure git to use HTTPS authentication
+        uses: GuillaumeFalourd/SSH-to-HTTPS@v1
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '17'
```

Comment:
I'm not sure if we should be using version 17 on this project; I'm currently not 100% sure what version is expected to be used, so maybe this is a bit preliminary